fixtures/webview.py

```diff
@@ -399,6 +399,16 @@ def rex_base_url(request):
         return base_url
 
 
+@pytest.fixture
```

Comment:
Should this test live in cnx-automation?

Reply:
You mean it should be in its own repo? Like cops-automation? How would I do it?